source/_components/sensor.fronius.markdown

```diff
@@ -0,0 +1,66 @@
+---
+layout: page
+title: "Fronius"
+description: "Instructions on how to connect your Fronius Inverter to Home Assistant."
+date: 2018-01-05 11:43:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Energy
+logo: fronius.png
+ha_iot_class: "Local Polling"
+ha_release: 0.92
+---
+
+The `fronius` sensor will poll a [Fronius](http://www.fronius.com/) solar inverter, battery system or smart meter and present the values as sensors (or attributes of sensors) in Home Assistant.
+
+To enable this sensor, add the following lines to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: fronius
+    resource: http://192.168.42.47
+    device: 1
+    sensor_type: inverter
+
+  - platform: fronius
+    resource: http://192.168.42.47
+    sensor_type: meter
+    scope: system
+```
+
+Configuration variables:
```

Comment:
Please use the [new format](https://developers.home-assistant.io/docs/en/documentation_create_page.html#configuration)